crates/stackable-webhook/src/tls/mod.rs

```diff
@@ -25,130 +22,89 @@ use tokio_rustls::{
 use tower::{Service, ServiceExt};
 use tracing::{Instrument, Span, field::Empty, instrument};
 use tracing_opentelemetry::OpenTelemetrySpanExt;
+use x509_cert::Certificate;
+
+mod cert_resolver;
 
-pub type Result<T, E = Error> = std::result::Result<T, E>;
+pub const WEBHOOK_CA_LIFETIME: Duration = Duration::from_minutes_unchecked(3);
+pub const WEBHOOK_CERTIFICATE_LIFETIME: Duration = Duration::from_minutes_unchecked(2);
+pub const WEBHOOK_CERTIFICATE_ROTATION_INTERVAL: Duration = Duration::from_minutes_unchecked(1);
```

Comment:
reminder to bump these before merging. Currently they are so low for easy testing

Reply:
done in e3e1119